components/View/FixedTabs/index.js

```diff
@@ -93,7 +95,9 @@ export default class ScrollTabs extends React.PureComponent {
                     data={tabList}
                     modifier={this.renderTab}
                 />
-                <div className={styles.void} />
+                <div className={styles.void}>
```

Comment:
Not a good way to do it, void is there to fill up the empty space, shouldn't be any empty element in there

Reply:
I know. I made it have two purposes. Fix gardeu na, kasari garna parne ho?